lib/src/util/bolt_module.cc

```diff
@@ -74,11 +74,8 @@ void BoltModule::processOutputAndUpdateMetadata(PXPAgent::ActionResponse &respon
                   response.prettyRequestLabel(), output);
         std::string execution_error {
                 lth_loc::format("The task executed for the {1} returned invalid "
-                                "UTF-8 on stdout - stderr:{2}",
-                                response.prettyRequestLabel(),
-                                (response.output.std_err.empty()
-                                 ? lth_loc::translate(" (empty)")
-                                 : "\n" + response.output.std_err)) };
```

Comment:
So a change here will be the error message will no longer attempt to print out stderr? Seems fine

Reply:
Yeah. Specifically there was a bug here where if both stdout _and_ stderr included invalid UTF-8, the result would include invalid UTF-8 because it included stderr without first validating it. I opted to just remove the stderr bit rather than adding that validation, on the basis that a) it seems likely that if stdout is invalid UTF-8, stderr probably is too, b) including stderr doesn't seem especially valuable in this case (since the task didn't _fail_, stderr isn't going to help explain what the problem was), c) I'm lazy.